misc/tools/install.sh

```diff
@@ -43,6 +43,8 @@ vz_git=https://github.com/volkszaehler/volkszaehler.org
 # cannot handle other hosts right now
 db_host=localhost
```

Comment:
Question to all developers ...
what's the matter with a foreing host?
I have changed this locally for my installation (existing DB on NAS) and I can't see why this should be a problem.
Only thing that might be interesting is to check for ``mysql-client`` instead of ``mysql`` package if there's a foreing MySQL host